internal/service/flexcluster/resource_test.go

```diff
@@ -13,19 +13,20 @@ import (
 )
 
 var (
-	resourceType   = "mongodbatlas_flex_cluster"
-	resourceName   = "mongodbatlas_flex_cluster.test"
-	dataSourceName = "data.mongodbatlas_flex_cluster.test"
+	resourceType         = "mongodbatlas_flex_cluster"
+	resourceName         = "mongodbatlas_flex_cluster.test"
+	dataSourceName       = "data.mongodbatlas_flex_cluster.test"
+	dataSourcePluralName = "data.mongodbatlas_flex_clusters.test"
 )
 
 func TestAccFlexClusterRS_basic(t *testing.T) {
 	tc := basicTestCase(t)
-	resource.ParallelTest(t, *tc)
+	resource.Test(t, *tc)
```

Comment:
Parallelisation of tests has been removed as this causes errors where pluralDS expects 1 flex cluster to be present but an additional cluster is found from the other Acc test.

Reply:
Would add as a comment so we have context in future reference

Reply:
Added comment